### ISSUE TITLE
Suggestion: Remove need for a seperate menu

### ISSUE THREAD

**Issue by Fabsian10000** [Ketheroth/Slots#3]
i just thought it might function a lot nicer if the added slots would instead be added to the normal inventory screen instead, that way you dont have to fumble around with seperate menus

**Comment by ink0stain4** [comment]
> i just thought it might function a lot nicer if the added slots would instead be added to the normal inventory screen instead, that way you dont have to fumble around with seperate menus

agreed. i feel iffy placing items in seperate inventories as it is.

**Comment by Fabsian10000** [comment]
A problem i might see popping up however is how it will affect other mods and potentially the way textures of the inventory will work. Mostly tho im thinking about, if the inventory would still be centered, things like tabs buttons in the inventory from other mods might be misplaced as a result of shifting the inventory around

**Comment by ink0stain4** [comment]
i see. Maybe another inventory tab, clickable at the top? I used to see a lot of mods like Baubles give you the option to scroll through inventory tabs.

**Comment by Fabsian10000** [comment]
Maybe you could get away with adding the additional slots to the bottom of the inventory without moving it around. Tho it might cause a problem if the gui scale is set to a bigger setting, this might be the best option as i rarely see other mods add functionality to the bottom of the screen. Other than that, a scroll bar might also do the trick i guess. Maybe both of those ideas could even be fused together, dynamically enabling a scroll bar whenever the expanded inventory wont fit the screen
